ladybug/datacollection.py

```diff
@@ -163,16 +162,27 @@ def get_highest_values(data, count):
             highest_values_index: Indicies of the n highest values in data 
                 list, ordered from highest to lowest
         """
-        lenght_data = len(data)
+        dataPoints = self._data
+        
+        values = list(obj._value for obj in dataPoints)
```

Comment:
`values = [obj._value for obj in dataPoints]` is better way of writing this.

Reply:
Also maybe we should add a property for this that only returns the values without dates and another one that only return dates. I'm commenting on these during the workshop and didn't get the chance to see if they are already there.